projects/packages/videopress/src/client/block-editor/blocks/video/components/player/types.ts

```diff
@@ -12,3 +16,11 @@ export type PlayerProps = {
 	preview: VideoPreviewProps;
 	isRequestingEmbedPreview: boolean;
 };
+
+export type NativePlayerProps = {
```

Comment:
At this point the native player props are divergent since we need some props from `edit.native` to pass to the `BlockCaption` component.

Reply:
Probably this is the first time we deal with types in the native version. I'm wondering if we could follow a similar approach as we do with components and have native-specific types in a separate file, i.e. `types.native.ts`, WDYT?

Reply:
Ah sure, I can move the type to a new file

Reply:
fixed in 3d1c471bd1335a7a8ba11d655a0e2154c858ab41